tools/include/eosio/abigen.hpp

```diff
@@ -729,6 +721,7 @@ namespace eosio { namespace cdt {
       public:
          explicit eosio_abigen_visitor(CompilerInstance *CI) {
             get_error_emitter().set_compiler_instance(CI);
+            set_suppress_ricardian_warning(ag.suppress_ricardian_warnings);
```

Comment:
I don't think you need this one. `suppress_ricardian_warnings` above is taken from `abigen`

Reply:
Yes, when I wrote this patch, `eosio_abigen_visitor` called `eosio_abigen_visitor::parse_contracts` instead of `abigen::parse_contracts`. This is obsolete now.

Reply:
got it. Yes, After some refactoring ricardian contracts generation was broken and recently it was fixed including this warning suppression.
So now other then correcting warning message there is nothing to do. Anyway if you'll add toolchain tests for this warning it will be helpful for future